integration

```diff
@@ -917,6 +917,7 @@
   "mletenay/home-assistant-goodwe-inverter",
   "mmillmor/geo_home",
   "modrzew/hass-flashforge-adventurer-3",
+  "moehrem/DiveraControl",
```

Comment:
As this is only relevant for users in a specific country, the released `hacs.json` file needs a country key.
<https://hacs.xyz/docs/publish/include>